### PR TITLE
 hwdb: add Linx Vision 8 rotation information

### PR DESCRIPTION
**Pull request by krissgjeng**

### CODE REVIEW

**Comment by krissgjeng** [comment]
Sorry about the extra whitespace that snuck in, i was copying the above lines and the extra whitespace got added somehow.

This is my first time trying to do a pull request.
Tell me if i have done something wrong.

File: hwdb.d/60-sensor.hwdb [hunk 1]
```diff
@@ -529,6 +529,10 @@ sensor:modalias:acpi:BOSC0200*:dmi:*:svnLINX*:pnLINX1010B:*
 sensor:modalias:acpi:KIOX000A*:dmi:*:svnLINX*:pnLINX12*64:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; 1, 0, 0; 0, 0, 1
 
+# Linx Vision 8
+sensor:modalias:acpi:KIOX000A*:dmi:*:svnLINX:pnVISION004:*
```
Comment (poettering) [inline hwdb.d/60-sensor.hwdb:530]: please merge this with the previous entry, i.e. place two `sensor:…` lines one after the other, and then one body for it... See the IdeaPad lines further up for an example

These devices identify in a very similar way, and have the same matrix, hence merging the entries makes a lot of sense

**Review by poettering — CHANGES_REQUESTED** [review]

File: hwdb.d/60-sensor.hwdb (hunk 1 above)
Comment (krissgjeng) [reply hwdb.d/60-sensor.hwdb:530]: Done.

**Pull request merged**